test/intl402/PluralRules/constructor-option-read-order.js

```diff
@@ -0,0 +1,53 @@
+/// Copyright 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-initializepluralrules
+description: Checks the order of option read.
+features: [Intl.NumberFormat-v3]
+includes: [compareArray.js]
+---*/
+
+let optionKeys = [
+    // Inside InitializePluralRules
+    "localeMatcher",
+    "type",
+    // Inside SetNumberFormatDigitOptions
+        "minimumIntegerDigits",
+        "minimumFractionDigits",
+        "maximumFractionDigits",
+        "minimumSignificantDigits",
+        "maximumSignificantDigits",
+        "roundingPriority",
+        "roundingIncrement",
+        "roundingMode",
+        "trailingZeroDisplay",
```

Comment:
This disagrees with https://github.com/tc39/ecma402/pull/811 .
```suggestion
        "roundingIncrement",
        "roundingMode",
        "roundingPriority",
        "trailingZeroDisplay",
```